### ISSUE TITLE
FR: Multiple symbols

### ISSUE THREAD

**Issue by Samreay** [brian-the-dev/python-tradingview-ta#43]
Hi Brian,

Unsure if Ive missed in the doco, but it would be great to utilise the API to get multiple symbols at once. 

Do you know if this would be possible at all?

Cheers

**Comment by brian-the-dev** [comment]
Hey @Samreay,

Yes, it's not implemented yet and is possible. But, all the symbols must have the same screener. I'll implement this later.

**Comment by Samreay** [comment]
Hey Brian, if you can point me in the right direction I'd be happy to make a PR for this tonight :)

**Comment by brian-the-dev** [comment]
I already implemented it. You can check the [get_multiple_analysis](https://github.com/brian-the-dev/python-tradingview-ta/tree/get_multiple_analysis) branch. I'm still testing it before merging it into the main branch. Note: no error checking yet.

Sample code:
```python
from tradingview_ta import *

x = get_multiple_analysis(screener="america", interval=Interval.INTERVAL_1_HOUR, symbols=["nasdaq:tsla", "nyse:docn"])

print(x["NASDAQ:TSLA"].summary)
print(x["NYSE:DOCN"].summary)

# {'RECOMMENDATION': 'SELL', 'BUY': 11, 'SELL': 9, 'NEUTRAL': 6}
# {'RECOMMENDATION': 'BUY', 'BUY': 11, 'SELL': 6, 'NEUTRAL': 9}
```

The code is slightly different from the usual `get_analysis()` (see example on readme). I think it's simpler than having to instantiate a class.

**Comment by brian-the-dev** [comment]
This feature has been released in [v3.2.4](https://pypi.org/project/tradingview-ta/3.2.4/). Please update using `pip install -U tradingview_ta`. You may close this issue if you don't have any further questions. Have a nice day!

Note: This isn't in the documentation yet. I'll add it soon.

**Comment by Samreay** [comment]
Cheers!
